website/docs/src/content/cloud/pricing.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Pricing & limits
+description: GenSX Cloud Platform pricing and usage information
+---
+
+# Pricing & limits
+
+GenSX Cloud offers flexible pricing tiers designed to scale with your needs, including a free tier for individuals.
+
+## Pricing tiers
```

Comment:
I think we discussed that if you wanted to add another member to your org, that automatically required you to upgrade to pro? Is that still the case? if so we should document it.

If that's not the case, I worry our free tier might be too generous.

Reply:
Fixed.